src/autocomplete.js

```diff
@@ -254,6 +261,8 @@ export default class Autocomplete extends Component {
     this.setState({
       focused: -1
     })
+
+    this.handleInputChange(event)
```

Comment:
Preco sa v `handeInputFocus` vola `handleInputChange`. Input sa predsa nezmenil. Preco sa to deje?

Reply:
<3